Symfony/CS/Fixer/PSR2/FunctionCallSpaceFixer.php

```diff
@@ -121,11 +121,13 @@ private function getFunctionyTokens()
     }
 
     /**
-     * Gets the name of tokens that are actually language construction.
+     * Gets the token kinds of actually language construction.
      *
+     * @staticvar int[] $languageConstructionTokens Token kinds.
```

Comment:
`@var` is bad too, I would just remove it, it is encapsulated in method itself and not needed in docs.